tools/parameter_tuning.py

```diff
@@ -0,0 +1,55 @@
+from sisyphus import Job, Task, tk
+from typing import Any, Literal, Sequence, Union
+
+import numpy as np
+
+from i6_core.util import instanciate_delayed
+
+
+class PickOptimalParametersJob(Job):
+    """
+    Pick a set of optimal parameters based on their assigned (dynamic) score value.
+    Each optimal parameter is outputted individually to be accessible in the Sisyphus manager.
+
+    Can be used to e.g. pick best lm-scale and prior scale to a corresponding ScliteJob.out_wer.
+    """
+
+    def __init__(
+        self,
+        parameters: Sequence[Sequence[Any]],
+        values: Sequence[tk.Variable],
+        mode: Union[Literal["maximize"], Literal["minimize"]],
```

Comment:
I like explicit keyword args.
```suggestion
        *,
        mode: Union[Literal["maximize"], Literal["minimize"]],
```
Or maybe put the `*,` even earlier.